meta-genivi-dev/recipes-sota/rvi-sota-client/rvi-sota-client_git.bb

```diff
@@ -112,10 +112,9 @@ SRC_URI[index.sha256sum] = "86114b93f1f51aaf0aec3af0751d214b351f4ff9839ba031315c
 
 SYSTEMD_SERVICE_${PN} = "sota_client.service"
 
-DEPENDS += " openssl "
+DEPENDS += " openssl dbus "
```

Comment:
OK, but I'm sure you can understand that we can't call something a fix without knowing how/why it works cause it could just be a race-condition/coincidence that patch *seems* to work and in reality it doesn't. That's why we need to know the specific reason. You've done most of the work already and mostly likely your patch is good so I'm hoping that you'll have time to find out and be able to provide a complete story in commit log.